docs/Upgrade.md

```diff
@@ -0,0 +1,45 @@
+# Upgrade
+
+## 1. Overview
+
+This document contains pointer to instructions on how to upgrade the
+Zonemaster::Backend component. An upgrade usually consist of an upgrade script
+to upgrade the database, and instructions to install new dependencies.
+
+
+## 2. Upgrading Zonemaster::Backend
+
+To upgrade Zonemaster::Backend perform the following tasks:
+
+  1. stop the `zm-rpcapi` and `zm-testagent` daemons
```

Comment:
In FreeBSD they are, for some reason, `zm_rpcapi` and `zm_testagent`

Reply:
Should we consider standardize how the daemons are called?

Reply:
Yes, but is a breaking change how it is called. It is better with hyphen than underscore.